plugins/ubuntu/readme.md

```diff
@@ -1,21 +1,50 @@
+| aga     | `sudo apt-get autoclean`                                               | Clears out the local reposityory of retrieved package files that can no longer be downloaded      | 
+| agb     | `sudo apt-get build-dep <source_pkg>`                                  | Installs/Removes packages to satisfy the dependencies of a specified build pkg                    | 
+| agc     | `sudo apt-get clean`                                                   | Clears out the local repository of retrieved package files leaving everything from the lock files | 
+| agd     | `sudo apt-get dselect-upgrade`                                         | Follows dselect choices for package installation                                                  | 
+| agi     | `sudo apt-get install <pkg>`                                           | Install the specified package                                                                     | 
+| agli    | `apt list --installed`                                                 | List the installed packages                                                                       | 
+| aglu    | `sudo apt-get -u upgrade --assume-no`                                  | Run an apt-get upgrade assuming no to all prompts                                                 | 
+| agp     | `sudo apt-get purge <pkg>`                                             | Remove a package including any configuration files                                                | 
+| agr     | `sudo apt-get remove <pkg>`                                            | Remove a package                                                                                  | 
+| ags     | `apt source <pkg>`                                                     | Fetch the source for the specified package                                                        | 
+| agu     | `sudo apt-get update`                                                  | Update package list                                                                               | 
+| agud    | `sudo apt-get update && sudo apt-get dist-upgrade`                     | Update packages list and perform a distribution upgrade                                           | 
+| agug    | `sudo apt-get upgrade`                                                 | Upgrade available packages                                                                        | 
+| agar    | `sudo apt-get autoremove`                                              | Remove automatically installed packages no longer needed                                          | 
+| aguu    | `sudo apt-get update && sudo apt-get upgrade`                          | Update packages list and upgrade available packages                                               | 
+| allpkgs | `dpkg --get-selections | grep -v deinstall`                            | Print all installed packages                                                                      | 
```

Comment:
The `|` needs to be escaped with backslash (`\|`) or it doesn't render correctly.